docs/develop/howtos/dev_email.md

```diff
@@ -0,0 +1,64 @@
+# Test e-mails in development
```

Comment:
minor/comment: you might need to add a reference this file in the `mkdocs.yaml` so that it appears in the side-menu.